packages/errors/src/context.ts

```diff
@@ -45,4 +34,15 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<{
         actualNonceValue: string;
         expectedNonceValue: string;
     };
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 27 to 36]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->The linter won't enforce this for the next time though, right, because it doesn't enforce sort order of ‘the variable names inside computed properties.’

If there's a linter that _does_, feel free to throw it into `@solana/eslint-config-solana`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2185?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
I did not find one, but this one seems interesting for other reasons.
https://typescript-eslint.io/rules/sort-type-constituents